src/frontends/pytorch/src/helper_ops/internal_op.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "openvino/frontend/decoder.hpp"
+#include "pt_framework_node.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+
+class InternalOpDecoder : public DummyDecoder {
+public:
+    explicit InternalOpDecoder(const std::string& op_type, const size_t num_outputs)
+        : m_op_type(op_type),
+          m_num_outputs(num_outputs) {}
+    const std::string& get_op_type() const override {
+        return m_op_type;
+    }
+    size_t num_of_outputs() const override {
+        return m_num_outputs;
+    }
+    size_t get_subgraph_size() const override {
+        return 0;
+    }
+    const std::string& decoder_type_name() const override {
+        return m_decoder_type;
+    }
+
+private:
+    const std::string m_op_type;
+    const std::string m_decoder_type = "internal_op";
+    const size_t m_num_outputs;
+};
+
+class InternalOperation : public PtFrameworkNode {
```

Comment:
`InternalOp` doesn't always correspond to a node in the original graph, why should we derive it from `PtFrameworkNode`?

Reply:
To avoid issues with reporting such operations when they are left in graph, otherwise that reporting should be reworked, which is out of scope of these changes